### DISCUSSION TITLE
群晖只能docker版本才能识别插件吗？

### DISCUSSION THREAD

**Discussion by PiPi-happy** [metatube-community/jellyfin-plugin-metatube#131] [General]
如题，官方套件和Emby官方下载套件安装，都无法识别插件。。。

**Comment by xjasonlyu** [comment]
只要版本符合要求就可以安装的

**Reply by PiPi-happy** [reply]
<img width="486" alt="image" src="https://user-images.githubusercontent.com/13795727/218660397-9fb4fc77-6b98-43bf-9176-522f0bb707fb.png">
安装的是4.7.10
也尝试过最新的4.7.11，都无法识别

**Reply by xjasonlyu** [reply]
那可能是你插件权限没给对吧

**Reply by PiPi-happy** [reply]
好的，谢谢，我再研究研究

**Reply by PiPi-happy** [reply]
解决了，根据emby日志，果然是权限问题，重新赋权给用户就行了

**Reply by wakloij66778** [reply]
大佬请问怎么给权限？我的是套件版4.7.14.0无法安装插件

2023-10-17 23:15:23.290 Error App: Error loading assembly /var/packages/EmbyServer/var/plugins/MetaTube.dll
    *** Error Report ***
    Version: 4.7.14.0
    Command line: /volume1/@appstore/EmbyServer/system/EmbyServer.dll -programdata /var/packages/EmbyServer/var -ffdetect /var/packages/EmbyServer/target/bin/ffdetect -ffmpeg /var/packages/EmbyServer/target/bin/ffmpeg -ffprobe /var/packages/EmbyServer/target/bin/ffprobe -nolocalportconfig -ignore_vaapi_enabled_flag -pidfile /var/packages/EmbyServer/var/EmbyServer.pid -defaultdirectory /volume1/Public -updatepackage emby-server-synology72_{version}_x86_64.spk -noautorunwebapp
    Operating system: Linux version 4.4.302+ (root@build5) (gcc version 12.2.0 (GCC) ) #69057 SMP Sat Sep 23 21:48:04 CST 2023
    Framework: .NET 6.0.20
    OS/Process: x64/x64
    Runtime: volume1/@appstore/EmbyServer/system/System.Private.CoreLib.dll
    Processor count: 2
    Data path: /var/packages/EmbyServer/var
    Application path: /volume1/@appstore/EmbyServer/system
    System.UnauthorizedAccessException: System.UnauthorizedAccessException: Access to the path '/var/packages/EmbyServer/var/plugins/MetaTube.dll' is denied.
